### ISSUE TITLE
android version 

### ISSUE THREAD

**Issue by xxcriticxx** [Jo2003/vlc_record#8]
@Jo2003 do you have android version of your program?

**Comment by Jo2003** [comment]
No. Also if it would work (which I haven't checked) touch might be a problem.

**Comment by xxcriticxx** [comment]
Apple touch on iPhone?

**Comment by zk140t** [comment]
any chance for updates? apple version? modern linux version?
